.github/workflows/release.yml

```diff
@@ -35,13 +35,6 @@ jobs:
             arch: arm64
     runs-on: ${{matrix.os}}
     steps:
-      - name: Install Rust Toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: ${{env.RUST_TOOLCHAIN}}
-          default: true
-          profile: minimal
-          target: wasm32-unknown-unknown
       - name: Check Out Repo
         uses: actions/checkout@v3
```

Comment:
This is all that is repeated in changes below, essentially base off toolchain action on stable and our rust-toolchain.toml will override it